phlib/treenew.c

```diff
@@ -1258,6 +1258,18 @@ VOID PhTnpOnKeyDown(
         return;
     if (PhTnpProcessNodeKey(Context, VirtualKey))
         return;
+
+    // handle standard key presses
+    switch (VirtualKey)
+    {
+    case 'A':
+        if (GetKeyState(VK_CONTROL) < 0)
+            TreeNew_SelectRange(hwnd, 0, -1);
+        return;
+    }
```

Comment:
@dmex is this okay to do with treenew? Any reason not to do this?

Reply:
I think it is common (expected) behavior of tree/list views to be able to select all items.

Most of them had explicit handling of `Ctrl+A`. Though, some didn't (maybe just by mistake?).

It will be still possible to _disable_ `Ctrl+A` for particular tree view (if not wanted) by explicit handling the event and marking it as handled (`keyEvent.Handled = true`).